job-service-acceptance-tests/pom.xml

```diff
@@ -374,36 +370,6 @@
                             </build>
                         </image>
 
-                        <!-- Image for Batch Worker plugins -->
-                        <image>
-                            <alias>plugin</alias>
-                            <name>${project.artifactId}-worker-batch-test-plugins:${project.version}</name>
-                            <build>
-                                <from>${projectDockerRegistry}/library/busybox</from>
-                                <assembly>
-                                    <exportTargetDir>true</exportTargetDir>
-                                    <basedir>/mnt/mesos/sandbox/batch-plugins</basedir>
-                                    <inline>
-                                        <dependencySets>
-                                            <dependencySet>
-                                                <includes>
-                                                    <include>com.hpe.caf.worker.batch:worker-batch-test-plugin</include>
-                                                </includes>
-                                                <outputDirectory>.</outputDirectory>
-                                                <useProjectArtifact>false</useProjectArtifact>
-                                            </dependencySet>
-                                        </dependencySets>
-                                        <fileSets>
-                                            <fileSet>
-                                                <directory>${project.build.directory}/worker-batch-plugins-package-unpacked/batch-plugins</directory>
-                                                <outputDirectory>.</outputDirectory>
-                                            </fileSet>
-                                        </fileSets>
-                                    </inline>
-                                </assembly>
-                            </build>
-                        </image>
-
```

Comment:
Removing this is intentional?

Reply:
Yes the worker-batch-plugins doesn't exist since quite a while, also it has references to HPE in the pom. Removing it has no harm to the tests.

![image](https://github.com/user-attachments/assets/24ef6f22-4a91-4d17-8960-88fb7f66e94b)